install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -726,10 +721,6 @@ data:
   # the buffer size is set to the default monitor queue size.
   hubble-event-queue-size: {{ .Values.hubble.eventQueueSize | quote }}
 {{- end }}
-{{- if hasKey .Values.hubble "flowBufferSize" }}
-  # DEPRECATED: this block should be removed in 1.11
-  hubble-flow-buffer-size: {{ .Values.hubble.flowBufferSize | quote }}
-{{- end }}
```

Comment:
same as this one.

Reply:
yep, same clean up mentioned in the commit message

Reply:
to give some background, I am doing piece meal to document missing helm options and helm clean up  starting from  https://github.com/cilium/cilium/pull/21195, the initial thought is to  add the missing options and clean up in one PR split in a few commits, but it turned out the change is too big, can be difficult to review and also result in rebase conflicts, kind of pain the in the neck, so it is suggested to do this by piece meal :)

Reply:
@aanm to be fair [I asked for this removal on the way](https://github.com/cilium/cilium/pull/21817#pullrequestreview-1149183370) since there was already a cleanup diff in the patch, so this one is on me. If you feel strongly I'm of course ok to have it done in another PR.

Reply:
@kaworu @vincentmli could we compromise and leave the changes on this PR but making them in a different commit, what do you think?

Reply:
fine by me :)